gluefactory/configs/COMMON_BENCHMARK_CONF.yaml

```diff
@@ -0,0 +1,71 @@
+model:  # TODO: REPLACE MODEL WITH YOU MODEL CFG YOU WANT TO BENCHMARK
+  name: two_view_pipeline
+  extractor:
+    name: extractors.joint_point_line_extractor
+    aliked_model_name: "aliked-n16"
+    max_num_keypoints: 2048  # setting for training, for eval: -1
```

Comment:
You mentioned a threshold on the score map instead of setting max_num_points right?

Reply:
yes, in theory we should set a threshold that returns enough good keypoints. As we don't have this yet and benchmarks are set to use a max num of kp, It should be fine for now to use ex the top 2048 keypoints

Reply:
I will open issue regarding fixing threshold later so we remember doing that but merge for now